source/sfp/1_water/sfp_115_nl_abiotiek_multimeter/01_afhankelijkheden.Rmd

```diff
@@ -1,5 +1,15 @@
 # Afhankelijkheden
 
+Procedures waarnaar in deze procedure verwezen wordt:
+
+-   SIP-055B: Veldmeter voor pH, EC en O2, WTW Multi 3430 (in opmaak)
+
+-   SVP-015: Bioveiligheidsmaatregelen (in opmaak)
+
+-   SVP-112: Veiligheid in en rond water (in opmaak)
+
+-   SVP-114: Bemonstering waterkolom oppervlaktewater
+
```

Comment:
Al deze tekst moet je inderdaad verwijderen. Afhankelijkheden worden automatisch in de tabel (zie code chunk met label dependencies) opgesomd als je deze toegevoegd hebt met `protocolhelper::add_dependencies`.
Enkel sfp-114-nl is gepubliceerd. Je kan de volgende code gebruiken om deze [dependency toe te voegen aan je protocol](https://github.com/inbo/protocolsource/blob/main/CONTRIBUTING.md#what-to-do-in-case-of-dependencies):

```
protocolhelper::add_dependencies(
    code_mainprotocol = "spp-115-nl",
    protocol_code = "sfp-114-nl",
    version_number = "2024.03"
)
```

Naar de niet gepubliceerde protocols kan je in de tekst zelf verwijzen zoals we dat in protocol sfp-114-nl hebben gedaan met syntax voor een weblink. Dus bv `[SIP-055B](hier de juist url)`.

Reply:
Moeten de procedures die verwijzen naar deze procedure ook opgesomd worden? In het originele document staan SPP-116: Abiotische staalname stilstaande wateren en SPP-117: Abiotische staalname stromende wateren vermeld. Maar ik kan begrijpen dat dit overbodig is.

Reply:
Ik krijg de volgende foutmelding als ik bovenstaande code in de console probeer te runnen:
"Error in get_path_to_protocol(code_mainprotocol) : 
  Do you want to get the path?
                 If so, check if the protocol code is correct.
                 If you want to set a path for a new protocol,
                 you need to provide a short title"

Reply:
Zo te zien heb ik in bovenstaande code een foutje gemaakt: het moet sfp-115-nl zijn in plaats van spp-115-nl